src/wtforms/fields/core.py

```diff
@@ -535,7 +535,7 @@ def process_formdata(self, valuelist):
 
     def pre_validate(self, form):
         for v, _ in self.choices:
-            if self.data == v:
+            if self.data == v or self.data is None:
```

Comment:
You can, even more, simplify this stuff like this  `if self.data == v or not self.data`.